packages/client/src/pages/common/nav/nav.tsx

```diff
@@ -1,24 +1,63 @@
-import React from "react";
-import { AppBar, Toolbar, Typography, Button,ButtonBase } from "@material-ui/core";
+import React, { Dispatch, SetStateAction, useState } from "react";
+import PostDialog from "../../home/action-group/post-dialog";
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  Button,
+  ButtonBase,
+} from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+import { useAppSelector } from "../../../redux/store";
+import ProfileButton from "./profile-button";
 
+// Using mui theme for consistent spacing
 const useStyles = makeStyles((theme: any) => ({
   root: {
     flexGrow: 1,
   },
-  menuButton: {
-    marginLeft: theme.spacing(2),
-  },
   iconContainer: {
     flexGrow: 1,
   },
-  profileButton: {
-    textTransform: "none",
-  }
+  menuButton: {
+    marginLeft: theme.spacing(2),
+  },
 }));
 
+function profileOptions(user: any, classes: any, setModalOpen: any) {
```

Comment:
ARE WE USING FUNCTION FUNCTIONS OR ARROW FUNCTIONS AAAAAAAAA

Reply:
MY BRAIN DEFAULTS TO FUNCTION FUNCTIONS AAAAAAAAAA